### ISSUE TITLE
Trying to remove watermarks from scanned documents:  Image size not supported

### ISSUE THREAD

**Issue by gpadvorac** [zuruoke/watermark-removal#35]
I'm removing watermarks from scanned documents and don't have control over the image size, and I'm getting this error:

image size: (2287, 1394, 3)
mask image size: (683, 1024, 3)
Image size not supported!!!

Is there a way to change this?  I didn't see anything in the docs about size limitations.

**Comment by fillwithjoy1** [comment]
No, not really. Unless you resize the image size to match the mask (which probably won't work), which is the only way I think you can get past this issue
